Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs

```diff
@@ -141,7 +141,35 @@ public IEnumerator BoundsOverrideTest()
         /// Uses near interaction to scale the bounds control by directly grabbing corner
         /// </summary>
         [UnityTest]
-        public IEnumerator ScaleViaNearInteration()
+        public IEnumerator FlickeringBoundsTest()
+        {
+            BoundsControl boundsControl = InstantiateSceneAndDefaultBoundsControl();
+            boundsControl.BoundsControlActivation = BoundsControlActivationType.ActivateByProximityAndPointer;
+            yield return VerifyInitialBoundsCorrect(boundsControl);
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+            
+            boundsControl.gameObject.transform.position = new Vector3(0, 0, 1.386f);
+            boundsControl.gameObject.transform.rotation = Quaternion.Euler(0, 45.0f, 0);
+            
+            TestHand hand = new TestHand(Handedness.Left);
+            yield return hand.Show(new Vector3(0, 0, 1));
+            
+            yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+            
+            // Check for a few loops that the hand is not flickering between states
+            int iterations = 15;
```

Comment:
Nit: When using arbitrary numbers it's best to leave a comment that it's an arbitrary number. One of the scariest things going back to old code is seeing uncommented arbitrary numbers and wondering "is it safe to change this? does this just need to be a non-zero number?" 

I mean there are plenty of scarier things, but it's a helpful note to leave. Yes it increases verbosity, the tradeoff I believe is worth it based on experiences I've seen (i.e. maybe a sleep(1000) actually was important because it was related to an animation timeout which was too hard to refactor in a way that could be consumed in test code)